src/index.js

```diff
@@ -43,12 +45,12 @@ const REST = 'rest'
  * @returns {Object} Object hash where each key is one of the given props whose
  * value is a react component matching that prop
  */
-export default function seapig (children, schema = {}) {
-  invariant(schema && typeof schema === 'object', 'schema must be an object')
```

Comment:
The `invariant` check is still useful because Flow only does static checks and we want to handle dynamic errors as well. 

Nothing is stopping the user from doing `seapig(this.props.children, null)` for example.

Reply:
Also, I just noticed the jsDoc above should be updated to something like:

> Object hash where each key is one of the given props concatenated with 'Children' and each value is a react component matching that prop.

Reply:
ah got it! I will add it back.